SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+We support any version published via our official distribution channels.
+
+## Reporting a Vulnerability
+
+Report potential vulnerabilities privately via [this form](https://github.com/Authenticator-Extension/Authenticator/security/advisories/new).
+Where appropriate, include a proof-of-concept and reproduction steps.
+We will provide an initial response within 5 days.
```

Comment:
For a purely free open source project, there is a lot of pressure to respond to any security report within 5 days. In fact, for non-urgent issues or issues that we do not think must be fixed, we can delay the response. During the holidays, when it is not convenient to work, it is very likely that we will not be able to respond within 5 days.

Reply:
"inital response" here means just acknowledging that we have received the issue. I can change it to "as soon as reasonably possible for the maintainers"?

Reply:
How do you think about "We will strive to provide an initial response within 5 days, though it is not guaranteed."

Reply:
That sounds appropriate to me.